packages/resolve-url-loader/index.js

```diff
@@ -147,6 +147,11 @@ function resolveUrlLoader(content, sourceMap) {
 
     // prepare the adjusted sass source-map for later look-ups
     sourceMapConsumer = new SourceMapConsumer(absSourceMap);
+  } else {
+    handleAsWarning(
+      'webpack misconfiguration',
+      'source-map not received'
```

Comment:
Please be more descriptive like "webpack or the upstream loader did not supply a source-map" 🤷‍♂️